libs/checkpoint-postgres/src/index.ts

```diff
@@ -12,15 +12,30 @@ import {
 } from "@langchain/langgraph-checkpoint";
 import pg from "pg";
 
-import { MIGRATIONS } from "./migrations.js";
+import { getMigrations } from "./migrations.js";
 import {
-  INSERT_CHECKPOINT_WRITES_SQL,
-  SELECT_SQL,
-  UPSERT_CHECKPOINT_BLOBS_SQL,
-  UPSERT_CHECKPOINT_WRITES_SQL,
-  UPSERT_CHECKPOINTS_SQL,
+  type SQL_STATEMENTS,
+  getSQLStatements,
+  getTablesWithSchema,
 } from "./sql.js";
 
+interface PostgresSaverOptions {
+  schema: string;
+}
+
+const _defaultOptions: PostgresSaverOptions = {
+  schema: "public",
+};
+
+const _ensureCompleteOptions = (
+  options?: Partial<PostgresSaverOptions>
+): PostgresSaverOptions => {
+  return {
+    ..._defaultOptions,
+    ...options,
+  };
+};
```

Comment:
Thanks for this. In the future you can also just in-line this, as it's a really common pattern for applying defaults. Having it be a well-named function call does express intent more clearly, however.